pkg/storj/path.go

```diff
@@ -19,3 +23,25 @@ func SplitPath(path Path) []string {
 func JoinPaths(paths ...Path) Path {
 	return strings.Join(paths, "/")
 }
+
+// CreatePath will create a Segment path
+func CreatePath(projectID uuid.UUID, segmentIndex int64, bucket, path []byte) (Path, error) {
```

Comment:
This sounds like it belongs more to satellite rather than in the general `pkg/storj` package.

Reply:
We don't have a satellite utils so not sure of a better place to move it

Reply:
moved back to metainfo